app/client-v2/src/components/ProjectView/NetworkGraph.vue

```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+import { ref } from "vue";
+import CytoscapeCanvas from "./CytoscapeCanvas.vue";
+const props = defineProps<{
+  graph: string;
+  cluster: string;
+}>();
+
+const fullScreenVisible = ref(false);
+</script>
+
+<template>
+  <Sidebar v-model:visible="fullScreenVisible" :header="`Cluster: ${props.cluster}`" position="full">
```

Comment:
This does feel like a weird use of a sidebar, but I guess it gets the job done. 
I wonder if there's accessibility issues... but then the network isn't exactly accessible anyway!